wremnants/syst_tools.py

```diff
@@ -1,20 +1,24 @@
 import hist
 import numpy as np
 
-def scale_helicity_hist_to_variations(scale_hist, sum_helicity=False, sum_ptV=False, rebinPtV=0):
-
+def scale_helicity_hist_to_variations(scale_hist, sum_helicity=False, sum_ptV=False, hasNoHelicity=False, rebinPtV=0):
+    # hasNoHelicity=True allows for usage with QCD scale histograms made with no helicity splitting (also no gen charge)
```

Comment:
Is this comment still correct/relevant now that the gen charge has been added to the non-helicity-split histogram?

Reply:
Yeah, have to remove the part about charge. The rest should stay to remember that "hasNoHelicity" is meant to use the histogram that does not have the helicity axis. Done just now